common/autoware_time_utils/CHANGELOG.rst

```diff
@@ -1,6 +1,24 @@
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This diff is something weird.

Reply:
Thanks for pointing out.
Fixed in 873a6bb497528e4ccf210182b1d6bc2cb1ba0eb1